compiler/src/dotty/tools/dotc/core/Symbols.scala

```diff
@@ -516,6 +516,18 @@ object Symbols {
     /** The current name of this symbol */
     final def name(implicit ctx: Context): ThisName = denot.name.asInstanceOf[ThisName]
 
+    def isHigherOrderTypeParameter(implicit ctx: Context): Boolean = this.maybeOwner.isTypeParameterOrSkolem
```

Comment:
It's just used once. I'd just use the RHS directly in Erasure. That would avoid having to look up what the term means.